### PR TITLE
add treasurehunter extension

### PR DESCRIPTION
**Pull request by MoPaMo**
### Project Details
1. **What is the name of your project?**
TreasureHunter

2. **How many hours did you spend developing this?**  

5h (but updates incoming)

3. **What problem does your extension solve, or what makes it unique?**  

Better goal setting and motivation for the High Seas program by adding a progress bar and goals/favourite bounties to the store, with a beautiful layout and easy installation while maintaining privacy.

4. **How many people have tested your extension?**  

4-5
5. **Does your extension work in both Chrome and Firefox?**  

Yes (and I've attached both extensions, not sure if thats right)
---

### Additional Comments
Original repo: https://github.com/MoPaMo/treasurehunter
I already pbllished to Mozilla addons: https://addons.mozilla.org/addon/treasurehunter/
Thanks for the cool project!

### CODE REVIEW

File: submission/treasurehunter/firefox/manifest.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "manifest_version": 2,
```
Comment (JosiasAurel) [inline submission/treasurehunter/firefox/manifest.json:2]: please use Manifest Version 3

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
Your extension must use the Manifest Version 3 to be accepted.

Also for some reason I can't run your extension in Chrome or Firefox. Please include instructions on how to run it in both browsers.

File: submission/treasurehunter/firefox/manifest.json (hunk 1 above)
Comment (MoPaMo) [reply submission/treasurehunter/firefox/manifest.json:2]: probably because of v3, i run it in brave but will check

**Comment by MoPaMo** [comment]
did you get any errors when adding it to chrome?

**Comment by JosiasAurel** [comment]
This is where I'm stuck. I can't click it to run 
image

**Comment by MoPaMo** [comment]
ah yeah, it just works out of the box once you open the highseas.hackclub.com domain

**Comment by JosiasAurel** [comment]
> ah yeah, it just works out of the box once you open the highseas.hackclub.com domain

please link a demo video as to how this works. This also seems to be too specific to high seas. ideally the extension should work out of it

**Comment by JosiasAurel** [comment]
closing due to inactivity

**Pull request closed without merging**